monai/data/csv_saver.py

```diff
@@ -29,20 +29,20 @@ class CSVSaver:
     def __init__(self, output_dir: str = "./", filename: str = "predictions.csv", overwrite: bool = True):
         """
         Args:
-            output_dir (str): output CSV file directory.
-            filename (str): name of the saved CSV file name.
-            overwrite (bool): whether to overwriting existing CSV file content. If we are not overwriting,
+            output_dir: output CSV file directory.
```

Comment:
The very first patch in this PR addresses that issue.  The typehints are injected into the documentation by sphinx.  It allows for a "DRY" (do not repeat yourself) paradigm so that the types are only defined once (which avoid errors).